napari/layers/surface/normals.py

```diff
@@ -33,7 +34,7 @@ class Normals(EventedModel):
 
     mode: NormalMode = Field(NormalMode.FACE, allow_mutation=False)
     visible: bool = False
-    color: ColorValue = 'black'
+    color: Union[str, Array[float, (3,)], Array[float, (4,)]] = 'black'
```

Comment:
this seems like a downgrade ... before, we could trust that `normal_mode.color` was an array... with this, won't all downstream consumers have to do `if isinstance(..., str)`?

Reply:
This is a merge error, definitely should have remained a `ColorValue`!